addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -21,11 +21,16 @@
             <field name="name">account.analytic.line.tree.hr_timesheet</field>
             <field name="model">account.analytic.line</field>
             <field name="arch" type="xml">
-                <tree editable="top" string="Timesheet Activities">
+                <tree editable="top" string="Timesheet Activities" js_class="timesheet_tree">
                     <field name="date"/>
                     <field name="name"/>
                     <field name="project_id" required="1" context="{'form_view_ref': 'project.project_project_view_form_simplified',}"/>
                     <field name="task_id" context="{'default_project_id': project_id}" domain="[('project_id', '=', project_id)]"/>
+                    <field
+                        name="timer"
+                        widget="timer_toggle_button"
+                        readonly='True'
```

Comment:
As truthy as ``readonly='0'``, and yet less readable. You can also inline small attributes like that ;)

As a reminder, we don't apply those pep8 rules (See: https://www.odoo.com/documentation/online/reference/guidelines.html):
E501: line too long
E301: expected 1 blank line, found 0
E302: expected 2 blank lines, found 1
E126: continuation line over-indented for hanging indent
E123: closing bracket does not match indentation of opening bracket's line
E127: continuation line over-indented for visual indent
E128: continuation line under-indented for visual indent
E265: block comment should start with '# '

Reply:
\* As truthy as `readonly="1"` ^^